web/src/components/network/NetworkPage.jsx

```diff
@@ -58,135 +43,117 @@ const NoWiredConnections = () => {
   );
 };
 
-/**
- * Internal component for displaying info when none WiFi connection is found
- * @component
- *
- * @param {object} props
- * @param {boolean} props.supported - whether the system supports scanning WiFi networks
- * @param {boolean} props.openWifiSelector - the function for opening the WiFi selector
- */
-const NoWifiConnections = ({ wifiScanSupported }) => {
-  const message = wifiScanSupported
-    ? _("The system has not been configured for connecting to a WiFi network yet.")
-    : _("The system does not support WiFi connections, probably because of missing or disabled hardware.");
-
-  return (
-    <Stack hasGutter>
-      <div>{_("No WiFi connections found.")}</div>
-      <div>{message}</div>
-    </Stack>
-  );
-};
-
 /**
  * Page component holding Network settings
  * @component
  */
 export default function NetworkPage() {
   const { network: client } = useInstallerClient();
-  const { connections: initialConnections, settings } = useLoaderData();
+  const { connections: initialConnections, devices: initialDevices, settings } = useLoaderData();
   const [connections, setConnections] = useState(initialConnections);
-  const [devices, setDevices] = useState(undefined);
-  const [selectedConnection, setSelectedConnection] = useState(null);
+  const [devices, setDevices] = useState(initialDevices);
+  const [updateState, setUpdateState] = useState(false);
```

Comment:
Is `updateState` used? Perhaps a better name could help to know the intention.

Reply:
it was updateNetworks, as the method was get from WifiSelector and updateDevicesConnections was like too much.. was not very inspired at the moment of writing it :P

Reply:
As discussed offline, it is wrong because we are not checking the `updateState` value.